bin/fmt

```diff
@@ -1,14 +1,14 @@
-#!/bin/sh
+#!/bin/bash
 
 set -eu
 
 # go list will exclude the vendor directory.  We use grep to exclude generated
 # code.
-dirs=$(go list -f \{\{.Dir\}\} ./... | grep -v controller/gen)
+read -ra dirs <<< "$(go list -f \{\{.Dir\}\} ./... | grep -v controller/gen)"
 # go list will list all subdirectories and goimports acts recursively.  This
 # results in certain files being reported multiple time.  Therefore, we must
 # dedup them.
-files=$(bin/goimports -l "$dirs" | sort -u)
+files=$(bin/goimports -l "${dirs[@]}" | sort -u)
```

Comment:
Perhaps there is no need to store the go result in the `dirs` variable, but have the `go list` command here on (current) line 11 instead?

Reply:
I don't think that would work, since we would run into the same problem where quoting the output would treat it as a single argument.  I believe that we need to use `read` to split the output of `go list` into an array.

Reply:
Either version (or the previous) seem not to work if the working path contains whitespace.

Reply:
Using

`mapfile -t files < <(bin/goimports -l "${dirs[@]}" | sort -u)`

will handle whitespace paths properly